packages/components/src/flex/README.md

```diff
@@ -22,6 +22,35 @@ const Example = () => {
 };
 ```
 
+The most common use-case for `<Flex />` would be to automatically align two (or more) child items of varying sizes. By default, `<Flex />` would vertically center align them, and evenly spread the inner content horizontally, starting at the edges.
+
+Child items can be added directly. However, it is recommended that they are wrapped with `<Flex.Item />` (which helps normalize layout styles).
+
+![Flex aligning and distributing 2 items of varying heights](https://make.wordpress.org/design/files/2020/06/flex-2-items.png)
+
+Adding more than 2 child items will spread them across with even gaps in between each item.
+
+![Flex aligning 3 items](https://make.wordpress.org/design/files/2020/06/flex-3-items.png)
+
+### Items and Blocks
+
+`<Flex />` provides 2 child components:
+
+-   `<Flex.Item />`
+-   `<Flex.Block />`
```

Comment:
Is it FlexItem  or Flex.Item...?

Reply:
Both are available. `Flex` has the `Item` and `Block` components available as static properties. From the `@wordpress/components` library, they're exported individually as `Flex`, `FlexItem`, and `FlexBlock`.

There were some discussions on static sub-components a while back when I had added `<Card />`.

Upon reflection. Seeing as how accessing sub-components via dot notation (or statics) isn't a pattern we have in the library. I'll update it to follow our current implementation